src/lib/libwasi.js

```diff
@@ -608,6 +608,21 @@ var WasiLibrary = {
     randomFill(HEAPU8.subarray(buffer, buffer + size));
     return 0;
   },
+
+  path_filestat_get__nothrow: true,
```

Comment:
Wrap these in `#if ALLOW_UNIMPLEMENTED_SYSCALLS` maybe?

Reply:
Sounds good! Let's see what the CI says...